athenet/layers/conv.py

```diff
@@ -127,8 +127,11 @@ def _get_output(self, layer_input):
         else:  # let Theano decide which implementation to use
             h, w = self.image_shape[0:2]
             pad_h, pad_w = self.padding
-            group_image_shape = (self.batch_size, n_group_channels,
-                                 h + 2*pad_h, w + 2*pad_w)
+            if self.batch_size is not None:
+                group_image_shape = (self.batch_size, n_group_channels,
+                                     h + 2*pad_h, w + 2*pad_w)
+            else:
+                group_image_shape = None
```

Comment:
Have a look at theano.tensor.nnet.conv.conv2d: image_shape is deprecated alias for input_shape. Should be renamed.